src/analysis/normed_space/lp_equiv.lean

```diff
@@ -12,10 +12,22 @@ import topology.continuous_function.bounded
 
 In this file we collect a variety of equivalences among various $$L^p$$ spaces.  In particular,
 when `α` is a `fintype`, given `E : α → Type u` and `p : ℝ≥0∞`, there is a natural linear isometric
-equivalence `lp_pi_Lpₗᵢ : lp E p ≃ₗᵢ pi_Lp p E`.
+equivalence `lp_pi_Lpₗᵢ : lp E p ≃ₗᵢ pi_Lp p E`. In addition, when `α` is a discrete topological
+space, the bounded continuous functions `α →ᵇ β` correspond exactly to `lp (λ _, β) ∞`. Here there
+can be more structure, including ring and algebra structures, and we implement these equivalences
+accordingly as well.
 
 We keep this as a separate file so that the various $$L^p$$ space files don't import the others.
 
+Recall that `pi_Lp` is just a type synonym for `Π i, E i` but given a different metric and norm
+structure, although the topological, uniform and bornological structures coincide definitionally.
+These structures are only defined on `pi_Lp` for `fintype α`, so there are no issues of convergence
+to consider.
+
+While `lp` is also a type synonym for `Π i, E i`, it allows for infinite index types. There is a
```

Comment:
I don't understand this sentence: don't you say that `lp` is precisely not a type synonym for `Π i, E i`, since there is an additional convergence condition?

Reply:
:facepalm: Yes, I was thinking about `pre_lp`, which *is* a type synonym. I'll edit this in a few minutes.